src/Components/App/App.js

```diff
@@ -1,23 +1,27 @@
 import './App.scss';
 import Header from '../Header/Header';
-import MoviePoster from '../MoviePoster/MoviePoster';
 import Homepage from '../Homepage/Homepage';
-import movieData from './movieData';
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 import MovieDetails from '../MovieDetail/MovieDetail';
 import ErrorPage from '../ErrorPage/ErrorPage';
+import getMovies from '../../apiCalls';
+
 function App() {
   const [movieDetails, setMovieDetails] = useState(false);
+  const [movies, setMovies] = useState([])
+  
+  useEffect(() => {
+   getMovies().then(data => setMovies(data))
+  }, [])
 
   const displayMovieDetails = id => {
-    const movie = movieData.movies.find(movie => movie.id === id);
-    setMovieDetails(movie);
+    getMovies(id).then(data => setMovieDetails(data))
   };
 
- const backToHomePage = () => {
-  setMovieDetails(false);
- }
- const error = true;
+  const backToHomePage = () => {
+    setMovieDetails(false);
+  };
+  const error = false;
```

Comment:
Could this error flag be stored in state? Kind of odd in react to have a floating variable like this.